go/src/glide.lock

```diff
@@ -552,7 +552,7 @@ imports:
 - name: github.com/koding/ec2dynamicdata
   version: 56f69f128b37c6bdfd61a6cac4b4b87e517fa2ee
 - name: github.com/koding/eventexporter
-  version: 8c411a9ed989426b2aa5bbcc6c9515cb3c76caf5
+  version: 0c3422129727d2c751c8a57e9570321b8d01217a
```

Comment:
Could you please update glide lock file. Commit `8c411a9ed989426b2aa5bbcc6c9515cb3c76caf5` is master's head in `eventexporter` repository.